data/schedule.yaml

```diff
@@ -1,36 +1,12 @@
 ---
 locations:
-  - location: Workshop Room 1
-    description: Security
-    events:
-      - name: SLOT FILLED
-        by: TBA
-        time: '09:30'
-      - name: SLOT FILLED
-        by: TBA
-        time: '11:30'
-      - name: SLOT FREE
-        by: NO ONE
-        time: '14:15'
-  - location: Workshop Room 2
-    description: Algorithms & Data Structures
-    events:
-      - name: SLOT FREE
-        by: NO ONE
-        time: '09:30'
-      - name: SLOT FREE
-        by: NO ONE
-        time: '11:30'
-      - name: SLOT FREE
-        by: NO ONE
-        time: '14:15'
   - location: Conference Room
     description: Main Track
+    live: true
```

Comment:
What does this tag mean?

Reply:
Not sure, cannot actually recall adding that. @butlerx any ideas?

Reply:
it was a way to say there would be a live stream. since replaced with checking for video link. so no longer needed

Reply:
okay so, just removed it.